planner/core/logical_plan_builder.go

```diff
@@ -3248,6 +3248,9 @@ func (b *PlanBuilder) buildUpdate(ctx context.Context, update *ast.UpdateStmt) (
 		if t.TableInfo.IsView() {
 			return nil, errors.Errorf("update view %s is not supported now.", t.Name.O)
 		}
+		if t.TableInfo.IsSequence() {
```

Comment:
Could we check it in preprocess.go?

Reply:
Yes, we can.
I handle it like what the view does.